### PR TITLE
Create Script to copy entire contents of Seattle Film Works disc regardless of year.

### PR DESCRIPTION
**Pull request by DeanGadberry**
I wrote these scripts in my efforts to save digital versions of photos from the 90s and early 2000s. 

I hope it can benefit someone else on the internet at some point.

### CODE REVIEW

**Comment by jasonacox** [comment]
Thanks for this @DeanGadberry !  I want to make a few minor changes to make it easier for users to customize for their system.

**Comment by DeanGadberry** [comment]
Sounds like a plan. Let me know how I can help. I’ve been using the script
this week to copy over hundreds of discs.

File: autosfw2jpg/README
Comment (jasonacox) [inline autosfw2jpg/README:1]: Can we rename this README.md so that Githgub formats in markdown?

File: autosfw2jpg/autosfw2jpg [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+ROLL_NUMBER=$1
+PICTURE_DATE=$2
+
+BLOCK_DEVICE=/dev/sr0
```
Comment (jasonacox) [inline autosfw2jpg/autosfw2jpg:6]: Let's put a comment here to help the user know where to update:

# Settings - Update these to match your system before running

**Review by DeanGadberry — COMMENTED** [review]
rename README to README.md

File: autosfw2jpg/autosfw2jpg-only-sfws [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ROLL_NUMBER=$1
+PICTURE_DATE=$2
+
+#clear workspace
+rm -f ~/tmp/conversions/*
+
+#Copy images from disk
+udisksctl mount -b $BLOCK_DEVICE
+let i=0
+for x in /media/dean/$ROLL_NUMBER/$ROLL_NUMBER.rol/*
```
Comment (jasonacox) [inline autosfw2jpg/autosfw2jpg-only-sfws]: I suggest we make `/media/dean/` a variable `$DISK` as with the other script.

```bash
for x in $DISK/$ROLL_NUMBER/$ROLL_NUMBER.rol/*
```

What do you think?

**Comment by jasonacox** [comment]
Hi @DeanGadberry  - let me know if you are ok with the changes I made and I'll merge this into the project.

Thanks for the great tool! 🙏

**Comment by DeanGadberry** [comment]
> Hi @DeanGadberry - let me know if you are ok with the changes I made and I'll merge this into the project.
> 
> Thanks for the great tool! 🙏

I appreciate the changes which you've made. This looks ready to me. I do think that the sfwjpg-only-sfws is repetitive, now that the other script handles those cases. Maybe we can remove that.

**Comment by jasonacox** [comment]
Thanks @DeanGadberry !  Great addition to the project.

**Pull request merged**